nix/default.nix

```diff
@@ -13,9 +13,8 @@ let
   nixpkgs-patched = bootstrap-pkgs.applyPatches {
```

Comment:
@nomeata I tried to short-cut the `nixpkgs-patched` by eliminating, but it didn't work out. I didn't invest too much time, though...

Reply:
You could copy what I did at https://github.com/dfinity/ic-hs/blob/master/nix/default.nix#L9, but it's not important